src/main/java/org/ibp/api/brapi/v2/inventory/LotDetails.java

```diff
@@ -0,0 +1,162 @@
+package org.ibp.api.brapi.v2.inventory;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import org.pojomatic.Pojomatic;
+import org.pojomatic.annotations.AutoProperty;
+
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+
+@AutoProperty
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class LotDetails {
+
+    private Double amount;
+
+    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ss.SSSXXX")
```

Comment:
Date format should be yyyy-MM-dd only. Based on this spec: https://github.com/plantbreeding/API/blob/master/Specification/GeneralInfo/Date_Time_Encoding.md